### DISCUSSION TITLE
Issue with starting and stopping container

### DISCUSSION THREAD

**Discussion by Solvaag** [lloesche/valheim-server-docker#8] [Q&A] [answered]
Hi,

This may be me being a total docker-noob, but I experienced that the container, when stopped and started, only booted up valheim-updater. This lead to the container running without warnings, but the server not actually going online. 

I logged into the container by using exec and starting it up manually, at which point the server was visible on the steam list again. 

Is this a mistake on my part? Should I always rm the image and re-mount it? 

Thanks for any input,

Solvaag

**Comment by lloesche** [comment, marked as answer]
This was a bug that has since been fixed here: https://github.com/lloesche/valheim-server-docker/commit/569c3c9d554ceb14963e274374cf4f74dc278e04

If you pull the latest Docker image it will work without having to `docker rm` the container.

**Reply by Solvaag** [reply]
Thank you!
